app/routes/agenda/agendaitems/agendaitem/news-item.js

```diff
@@ -35,10 +32,6 @@ export default class NewsitemAgendaitemAgendaitemsAgendaRoute extends Route {
     super.setupController(...arguments);
     const agendaitem = this.modelFor('agenda.agendaitems.agendaitem');
     controller.set('agendaitem', agendaitem);
-
-    const agendaItemTreatment = await agendaitem.get('agenda-item-treatment');
-    controller.set('agendaItemTreatment', agendaItemTreatment);
-
```

Comment:
Pretty sure this was incorrect, but since the controller value was never used we never saw this.
removed it.